presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMultimapAggAggregation.java

```diff
@@ -130,4 +205,29 @@ public void testDoubleRowMap()
 
         assertAggregation(aggFunc, map.isEmpty() ? null : map, builder.build());
     }
+
+    private static <K, V> void testMultimapAggWithGroupBy(InternalAggregationFunction aggFunction, GroupedAccumulator groupedAccumulator, int groupId, Type keyType, List<K> expectedKeys, Type valueType, List<V> expectedValues)
```

Comment:
nit: line too long

```
    private static <K, V> void testMultimapAggWithGroupBy(
            InternalAggregationFunction aggFunction, 
            GroupedAccumulator groupedAccumulator, 
            int groupId, 
            Type keyType, 
            List<K> expectedKeys, 
            Type valueType, 
            List<V> expectedValues)
```

Reply:
Should we add a checkstyle check for this? http://checkstyle.sourceforge.net/config_sizes.html#LineLength